cloud/aws/deploy/policy.go

```diff
@@ -86,6 +86,22 @@ var awsActionsMap map[resourcespb.Action][]string = map[resourcespb.Action][]str
 	resourcespb.Action_WebsocketManage: {
 		"execute-api:ManageConnections",
 	},
+	resourcespb.Action_QueueSend: {
+		"sqs:SendMessage",
+	},
+	resourcespb.Action_QueueReceive: {
+		"sqs:ReceiveMessage",
+		"sqs:DeleteMessage",
+	},
+	// XXX: Cannot be applied to single resources
+	// v1.Action_QueueList: {
+	// 	"sqs:ListQueues",
+	// },
```

Comment:
Can this be deleted?

Reply:
This was mostly to document out inability to apply that permission at the resource level, we have similiar comments for our other resources, did we want to clean them all out?